topcoffea/modules/YieldTools.py

```diff
@@ -2,6 +2,7 @@
 import json
 import pickle
 import numpy as np
+import coffea
```

Comment:
Do we need to import all of `coffea` here, or just `hist`?

Reply:
I added that import for [this check](https://github.com/TopEFT/topcoffea/blob/Data-MC/topcoffea/modules/YieldTools.py#L180) that I added to `get_cat_lables()`.

Reply:
Ok, thanks. I didn't catch that line at first!